### DISCUSSION TITLE
"Error on input validation"

### DISCUSSION THREAD

**Discussion by lucamitolo** [globaleaks/globaleaks-whistleblowing-software#3738] [Q&A]
Hi all,
I've installed and configured globaleaks on a VM and everythings seems alright, but when i try to insert a report, I get "Error on input validation" and no email wouldn't sent to any receipient.
How can I solve this?

**Comment by evilaliv3** [comment]
Hello @lucamitolo 

Do you have any error in your /var/globalealeaks/log/globaleaks.log ?

**Comment by vittorioluigidiana** [comment]
Hi everyone, i also encountered a similar error.

In my case i configured the **parent app** and two **child applications** (sites with hostname)
When i insert a report from the parent app, it succeeds. Trying to enter the system as a **Recipient** user i see the report with its details.

However, if i try to insert a report from a child app i encounter the following error:
**_"Error!
Error on input validation [Unable to deliver the submission to at least one recipient]"_**

The child app also has a **Recipient** user configured and who has already logged in for the first time.

No errors are reported in the file **_/var/globalealeaks/log/globaleaks.log_**

@evilaliv3 can you help me please?

**Comment by paolotomiato** [comment]
It also gives me and others the same error and doesn't send the report.
It happens with both the default server and the personal one.
The test email arrives instead.
There is nothing in the logs.

This is the message from the console when you try to send a fake report:
Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' 'sha256-pru43GdcNLwb4MwzOriCI9/9cKBzE5xeoLWHlKai1As='". Either the 'unsafe-inline' keyword, a hash ('sha256-uD+9kGdg1SXQagzGsu2+gAKYXqLRT/E07bh4OhgXN8Y='), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present

Has anyone managed to solve the same problem?

**Reply by paolotomiato** [reply]
You continue to receive reports of companies trying to use the platform and encountering the same problem

**Comment by paolotomiato** [comment]
From the latest tests carried out I could hazard a hypothesis: if I create the receiving user during the initial wizard it works while if I create it later it returns the error.

Can any developers give feedback?

**Comment by evilaliv3** [comment]
@paolotomiato : ensure that your reporting channels have always assinged at least one recipient.

When you run the wizard, the recipient that is created is automatically assigned to the default channel.
